### PR TITLE
SSG RHEL6 STIG alignment

### PR DESCRIPTION
**Pull request by redhatrises**
- Add some of the missing XCCDF rules from the DISA RHEL6 STIG
- Add mount_options.csv for OVAL template generation
- Add OVAL and Fixes content for new XCCDF rules

### CODE REVIEW

File: RHEL/6/xccdf/services/nfs.xml [hunk 1]
```diff
@@ -437,6 +437,28 @@ viewed or edited by an unauthorized user.
...
+To verify all squashing has been disabled, run the following command:
+<pre>$ grep all_squash /etc/exports</pre>
+</ocil>
+<rationale>
+The all_squash option maps all client requests to a single anonymous
+uid/gid on the NFS server, negating the ability to track file access
+by user ID.
+</rationale>
+<oval id="no_all_squash_exports" />
+<ident cce="TBD" />
```
Comment (shawndwells) [inline RHEL/6/xccdf/services/nfs.xml]: Needs CCE assignment

File: RHEL/6/xccdf/services/ssh.xml [hunk 2]
```diff
@@ -104,6 +104,28 @@ possibility of unauthorized access to the system.
...
+To check if PrintLastLog is enabled or set correctly, run the
+following command:
+<pre>$ sudo grep PrintLastLog /etc/ssh/sshd_config</pre>
+If configured properly, output should be <pre>yes</pre>
+</ocil>
+<rationale>
+Providing users feedback on when account accesses last occurred facilitates user
+recognition and reporting of unauthorized account use.
+</rationale>
+<ident cce="TBD" />
```
Comment (shawndwells) [inline RHEL/6/xccdf/services/ssh.xml]: Needs CCE.

File: RHEL/6/xccdf/system/auditing.xml [hunk 3]
```diff
@@ -222,6 +222,34 @@ normally.</i>
 <value selector="halt">halt</value>
 </Value>
 
+<Value id="var_auditd_space_left" type="number" >
+<title>Size remaining in disk space before prompting space_left_action</title>
+<description>The setting for space_left (MB) in /etc/audit/auditd.conf</description>
+<value selector=""></value>
```
Comment (shawndwells) [inline RHEL/6/xccdf/system/auditing.xml]: How will this variable be populated?

File: RHEL/6/xccdf/system/auditing.xml [hunk 4]
```diff
@@ -414,6 +442,33 @@ is used, running low on space for audit records should never occur.
...
+substituting <i>SIZE_in_MB</i> appropriately:
+<pre>space_left = <i>SIZE_in_MB</i></pre>
+Set this value to the appropriate size in Megabytes cause the system to
+notify the user of an issue.
+</description>
+<ocil clause="the system is not configured a specfic size in MB to notify administrators
+of an issue">
+Inspect <tt>/etc/audit/auditd.conf</tt> and locate the following line to
+determine if the system is configured correctly:
+<pre>space_left <i>SIZE_in_MB</i>`</pre>
```
Comment (shawndwells) [inline RHEL/6/xccdf/system/auditing.xml]: Erroneous ``'``

File: RHEL/6/xccdf/system/auditing.xml [hunk 5]
```diff
@@ -414,6 +442,33 @@ is used, running low on space for audit records should never occur.
...
+</ocil>
+<rationale>
+Notifying administrators of an impending disk space problem may allow them to
+take corrective action prior to any disruption.
+</rationale>
+<platform idref="cpe:/a:machine"/>
+<ident cce="27239-3" />
```
Comment (shawndwells) [inline RHEL/6/xccdf/system/auditing.xml]: Was this taken from the available pool or copy/pasted?

File: RHEL/6/xccdf/system/auditing.xml [hunk 6]
```diff
@@ -437,6 +492,72 @@ administrators of the system, who can take appropriate action.</rationale>
...
+switch to single-user mode, execute a script,
+or halt when the disk is out of space:
+<pre>disk_full_action single</pre>
+</ocil>
+<rationale>
+Taking appropriate action in case of a filled audit storage volume will minimize
+the possibility of losing audit records.
+</rationale>
+<platform idref="cpe:/a:machine"/>
+<ident cce="TBD" />
```
Comment (shawndwells) [inline RHEL/6/xccdf/system/auditing.xml]: Needs CCE

File: RHEL/6/xccdf/system/auditing.xml [hunk 7]
```diff
@@ -437,6 +492,72 @@ administrators of the system, who can take appropriate action.</rationale>
...
+switch to single-user mode, execute a script,
+or halt when the disk errors:
+<pre>disk_error_action single</pre>
+</ocil>
+<rationale>
+Taking appropriate action in case of disk errors will minimize the possibility of
+losing audit records.
+</rationale>
+<platform idref="cpe:/a:machine"/>
+<ident cce="TBD" />
```
Comment (shawndwells) [inline RHEL/6/xccdf/system/auditing.xml]: Needs CCE

File: RHEL/6/xccdf/system/auditing.xml [hunk 8]
```diff
@@ -735,6 +856,31 @@ audited.</rationale>
...
+<ocil clause="any are more permissive">
+Run the following command to check the mode of the system audit logs:
+<pre>$ sudo ls -ld /var/log/audit</pre>
+Audit log directories must be mode 0700 or less permissive.
+</ocil>
+<rationale>
+If users can write to audit logs, audit trails can be modified or destroyed.
+</rationale>
+<platform idref="cpe:/a:machine"/>
+<ident cce="TBD" />
```
Comment (shawndwells) [inline RHEL/6/xccdf/system/auditing.xml]: Needs CCE

File: RHEL/6/xccdf/system/software/gnome.xml [hunk 9]
```diff
@@ -336,6 +336,32 @@ and will also free up resources utilized by an idle session.
...
+If properly configured, the output should be <tt>&lt;Control&gt;&lt;Alt&gt;l</tt>.
+</ocil>
+<rationale>
+The ability to lock graphical desktop sessions manually allows users to
+easily secure their accounts should they need to depart from their workstations
+temporarily.
+</rationale>
+<platform idref="cpe:/a:machine"/>
+<oval id="gconf_gnome_screen_locking_keybindings" />
+<ident cce="TBD" />
```
Comment (shawndwells) [inline RHEL/6/xccdf/system/software/gnome.xml]: Nees CCE

File: RHEL/6/xccdf/system/software/integrity.xml [hunk 10]
```diff
@@ -158,7 +159,38 @@ Bugzilla #1277603.</b>
...
+is expected by the RPM database:
+<pre>$ rpm -Va | grep grep "^.....\(U\|.G\)"</pre>
+</ocil>
+<rationale>
+Ownership of binaries and configuration files that is incorrect
+could allow an unauthorized user to gain privileges that they should
+not have. The ownership set by the vendor should be maintained. Any
+deviations from this baseline should be investigated.
+</rationale>
+<ident cce="TBD" />
```
Comment (shawndwells) [inline RHEL/6/xccdf/system/software/integrity.xml]: Needs CCE

File: RHEL/6/xccdf/system/software/sudo.xml [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+The command should return no output.
+</ocil>
+<rationale>
+Without re-authentication, users may access resources or perform tasks for which they
+do not have authorization.
+<br /><br />
+When operating systems provide the capability to escalate a functional capability, it
+is critical that the user re-authenticate.
+</rationale>
+<ident cce="TBD" />
```
Comment (shawndwells) [inline RHEL/6/xccdf/system/software/sudo.xml]: Needs CCE

**Comment by redhatrises** [comment]
@shawndwells I believe that this is now ready for your re-review

**Pull request merged**